src/app/style/index.css

```diff
@@ -0,0 +1,7 @@
+@import '~semantic-ui-css/semantic.min.css';
```

Comment:
This has the side effect of including 624KB\* of styles before the
content of the page. This means that no content can be rendered until
we’ve successfully downloaded, parsed, and interpreted the CSS necessary
to set up the sprite sheets for the flag of Christmas Island. This will
incur significant latency on lower-end connections and hardware, so it’s
definitely something that we want to avoid.

The long-term solution is to simply include the stylesheet by URL
instead of inlining it. (Inlining our previous version of `index.css`
was okay because that stylesheet was tiny by design, and was not
intended to grow.) I’m tentatively okay with merging this as is if you
can add a TODO and make an issue for it. What do you think?

\* measured with:

```
curl --silent http://localhost:4000/ | sed '0,/<style>/d; /^<body /q' | wc -c
```